examples/android-room/build.gradle.kts

```diff
@@ -0,0 +1,91 @@
+plugins {
+    id("com.android.application")
+    kotlin("android")
+    kotlin("kapt")
+    id("io.sentry.android.gradle")
+}
+
+// useful for local debugging of the androidx.sqlite lib
+// make sure to checkout the lib from https://github.com/androidx/androidx/tree/androidx-main/sqlite/sqlite-framework
+//configurations.all {
+//    resolutionStrategy.dependencySubstitution {
+//        substitute(module("androidx.sqlite:sqlite-framework")).using(project(":sqlite-framework"))
+//    }
+//}
+
+android {
+    compileSdk = 30
+    defaultConfig {
+        minSdk = 21
+        targetSdk = 30
+        versionCode = 1
+        versionName = "1.0"
+    }
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = true
+            proguardFiles.add(getDefaultProguardFile("proguard-android-optimize.txt"))
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+    }
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_11.toString()
+    }
+}
+
+object Deps {
```

Comment:
Do you need this here? Can't you merge it inside the `buildSrc`?

Reply:
I can, but I wanted to keep it clean and not pollute with samples' stuff

Reply:
Extracted into a `Samples` object under the buildSrc